libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/gpg45/Gpg45ProfileEvaluator.java

```diff
@@ -76,8 +80,12 @@ public Optional<JourneyResponse> getJourneyResponseForStoredContraIndicators(
                         .with(LOG_CI_SCORE.getFieldName(), ciScore));
 
         if (ciScore <= Integer.parseInt(configService.getSsmParameter(CI_SCORING_THRESHOLD))) {
+            ipvSession.setCiFail(false);
```

Comment:
I think, we cal also add logs to the places where we set the 'ciFail' field to true, and we can assign an appropriate error code to it. Tracking this through the application logs in Splunk or CloudWatch would be easy.